src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -993,8 +993,11 @@ void Instruction::execute() {
         return executionNYI();
         break;
       }
-      case Opcode::AArch64_BRK: {
-        return executionNYI();
+      case Opcode::AArch64_BRK: {  // brk #imm16
```

Comment:
We need a case in the exception handler to support this, namely just a "unsupported breakpoint exception" printout